### ISSUE TITLE
Please export standard CommonJS module in the `main` field of package.json

### ISSUE THREAD

**Issue by fengyuanchen** [tbremer/postcss-atrule-bem#2]
```js
var postcss = require('postcss');
var atRuleBem = require('postcss-atrule-bem');

postcss([ 
  atRuleBem() // Error! It is not a function.
  atRuleBem.default() // Works, but it is ES6 module, not CommonJS module.
]);
```

**Comment by tbremer** [comment]
Thanks for this, didn't realize this was happening. Will fix as a breaking change.

**Comment by tbremer** [comment]
This was fixed with the 3.0.0 release that's now available on [npm](https://www.npmjs.com/package/postcss-atrule-bem).
